airbyte-integrations/connectors/source-stripe/main.py

```diff
@@ -5,9 +5,13 @@
 
 import sys
 
-from airbyte_cdk.entrypoint import launch
+from airbyte_cdk.entrypoint import AirbyteEntrypoint, launch
```

Comment:
All changes to source-stripe will be removed from this PR. This is only to show how the usage will look like